### ISSUE TITLE
I can't run studio-controls on Debian11

### ISSUE THREAD

**Issue by oscarsantis** [ovenwerks/studio-controls#79]
When i type "studio-controls on command line this is the result:
Traceback (most recent call last):
  File "/usr/local/bin/studio-controls", line 9, in <module>
    import jack
ModuleNotFoundError: No module named 'jack'

I have install Jack on may system and I run it normally with Qjackctl

**Comment by oscarsantis** [comment]
I've installed python-jack-client and the interface was loaded, but Jack didn't start.

**Comment by eeickmeyer** [comment]
You actually want python3-jack-client.

**Comment by oscarsantis** [comment]
Yes, I forgot to put the number 3 after python in the previous message

**Comment by eeickmeyer** [comment]
Here's the list of dependencies required to run studio-controls on a Debian-based system:
- a2jmidid
- jackd2
- pulseaudio-module-jack
- python3-alsaaudio
- python3-dbus
- python3-gi
- python3-jack-client
- zita-ajbridge

Beyond those, you need these for extra functionality:
- qasmixer
- qmidinet
- zita-njbridge

**Comment by oscarsantis** [comment]
I installed all the libraries, however the server startup and shutdown is much slower than in Ubuntustudio. Also, when I restart the computer, Jack is always turned off.
